src/Stratis.Bitcoin.Features.Consensus/PowConsensusValidator.cs

```diff
@@ -331,9 +376,20 @@ protected virtual void CheckMaturity(UnspentOutputs coins, int nSpendHeight)
             this.logger.LogTrace("(-)");
         }
 
-        public virtual void CheckInputs(Transaction tx, UnspentOutputSet inputs, int nSpendHeight)
+        /// <summary>
+        /// Checks that transaction's inputs are valid.
+        /// </summary>
+        /// <param name="tx">The transaction.</param>
```

Comment:
as above, i like more if the thing is a full sentence. 
`Transaction to check.` would feel better